addons/l10n_se/data/account_tax_template.xml

```diff
@@ -1143,5 +1143,63 @@
                     'account_id': ref('a2634')
                 })]"/>
         </record>
+
+        <!--Tax template in triangular trade-->
+        <record id="sale_tax_goods_TREC" model="account.tax.template">
+            <field name="chart_template_id" ref="l10nse_chart_template"/>
+            <field name="name">Trepartshandel - Momsfri försäljning av varor EU</field>
+            <field name="description">STE0</field>
+            <field name="amount">0</field>
+            <field name="type_tax_use">sale</field>
+            <field name="tax_group_id" ref="tax_group_0"/>
+            <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'base',
+                    'plus_report_line_ids': [ref('tax_report_line_38')]
+                }),
+                (0, 0, {
+                    'factor_percent': 100,
+                    'repartition_type': 'tax'
```

Comment:
Would not an account_id be in handy here as well?  @JulienAlardot

Reply:
I checked other localizations to match their way of doing it and I'll add the missing VAT % taxes for that (with account_id) but the 0% has not account linked to it so it has to stay that way (other loc do the same)